### PR TITLE
[iOS]: Specify FBSDK version in Podfile, and list version in code example

### PR DESCRIPTION
**Pull request by dacharyc**
### Staged Changes (Requires MongoDB Corp SSO)

- [Authenticate Users -> Facebook User](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/specify-FBSDK-version/sdk/ios/examples/authenticate-users/#facebook-user)

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

**Review by nathan-contino-mongo — APPROVED** [review]
LGTM

File: examples/ios/RealmExamples.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -18,8 +17,8 @@
 		4803B333251068F900CCAF97 /* LaunchScreen.storyboard in Resources */ = {isa = PBXBuildFile; fileRef = 4803B331251068F900CCAF97 /* LaunchScreen.storyboard */; };
 		48087CA2258800A600E19363 /* ReadWriteData.swift in Sources */ = {isa = PBXBuildFile; fileRef = 48087CA1258800A600E19363 /* ReadWriteData.swift */; };
 		4813483025C21948006209FC /* Notifications.m in Sources */ = {isa = PBXBuildFile; fileRef = 4813482F25C21948006209FC /* Notifications.m */; };
-		4813483B25C244CA006209FC /* (null) in Sources */ = {isa = PBXBuildFile; };
-		4815C5FD25C324F20014538D /* (null) in Sources */ = {isa = PBXBuildFile; };
+		4813483B25C244CA006209FC /* BuildFile in Sources */ = {isa = PBXBuildFile; };
+		4815C5FD25C324F20014538D /* BuildFile in Sources */ = {isa = PBXBuildFile; };
```
Comment (cbush) [inline examples/ios/RealmExamples.xcodeproj/project.pbxproj:21]: Are we sure the changes to this file are needed?

**Review by cbush — APPROVED** [review]
Looks good - thanks!

File: examples/ios/RealmExamples.xcodeproj/project.pbxproj (hunk 1 above)
Comment (dacharyc) [reply examples/ios/RealmExamples.xcodeproj/project.pbxproj:21]: No, I'm not sure. I looked at the diff before adding it to the commit and it mostly seemed to be moving stuff, but I wasn't sure if there's a best practice around committing or not committing this file under those circumstances. Should I remove it from this PR, @cbush ?

Comment (dacharyc) [reply examples/ios/RealmExamples.xcodeproj/project.pbxproj]: Removed it.

**Pull request merged**